userguide/aws/how-tos/maintain-a-cloud-formation-system.adoc

```diff
@@ -0,0 +1,273 @@
+= How to Mantain a CloudFormation System
+
+
+.
+
+== TroubleShooting
+
+This secion discusses things that could go wrong during installation or beyond, and provides some suggestions for dealing with them.
+
+=== Stack Creation
+
+There are several possible issues that could arise when starting the stack from CloudFormation (CFN). The goal of this section
+is to provide some tips debugging the issues, and knowledge about where to find various logs. Also keep in mind
+that starting the CFN stack will typically take in the order of 15-20 minutes, mostly because of the time it takes to setup the
+RDS cluster, so be patient...
+
+
+Let's start with the https://console.aws.amazon.com/cloudformation[CloudFormation console]: The stack will have a status which will be `CREATE_IN_PROGRESS` for the duration of the initialization, and will then either transition to `CREATE_COMPLETE` or `CREATE_FAILED`. In case of `CREATE_FAILED`, try to locate from the `Resources` tab which resource failed to initialize properly.
+
+=== Possible Issues
+
+The main possible issues can be summarized as:
+
+**Issues with the template**
+
+If you think there is an issue with the CFN template itself, please report any issue to `support@killbill.io`
+
+**AWS Errors**
+
+Typical Issues are:
+
+  * Insufficient IAM Permissions
+  * Limit Exceeded
+  * Security Group Does Not Exist in VPC
+  * RDS Cluster failed to come up
+  * ...
+
+Make sure to check the discussion above to ensure you have setup a valid VPC with valid subnets. Fow AWS specific issues, please refer to the https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html[AWS troubleshooting documentation]
+
+**Service Unavailable**
+
+We suggest to check the following:
```

Comment:
I left a question on each of these question on purpose: I know there is a lot of good info below in this doc, but, I think the way the doc is structured is such that we ask some good questions and then dump info but not necessarily clear to find the answer to the specific questions. Can we make it easier?